### DISCUSSION TITLE
Problem Finding Git LFS Files when Streamlit app is deployed

### DISCUSSION THREAD

**Discussion by oznerm** [git-lfs/git-lfs#4780] [Q&A]
I am running Git LFS for .csv files.  I currently have a 800MB data frame from which I want to make different plots. When I run the code in the localhost I have no problems (the file is located in a directory of the repository).  However when I run the public version of the app (on the web) I get the following error message:  FileNotFoundError: This app has encountered an error. The original error message is redacted to prevent data leaks. Full error details have been recorded in the logs.

Please advise.  Thanks!

**Comment by bk2204** [comment]
Hey,

I unfortunately don't know anything about Streamlit, but some Googling does indicate that it should support Git LFS.  You'd probably want to look at the logs to see what's going on and try to get some more information.

**Comment by oznerm** [comment]
Thanks for your response Brian.  Will check those logs.

**Reply by Suryam-Shaurya** [reply]
any solutions?
